assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -141,7 +179,13 @@ function Media3pPane(props) {
               onClick={onProviderTabClick}
             />
           </ProviderTabSection>
-          <CategorySection>{__('Coming soon', 'web-stories')}</CategorySection>
+          <CategorySection>
+            {fakePillList.map((e, i) => (
+              <Pill isSelected={e == selectedPill} key={i}>
+                {e}
```

Comment:
Can we have this as a prop of pill like `title`?

Reply:
Why do you think the body shouldn't be passed as children?

Reply:
As we already have a custom component Pill, I think it would be better if it would handle the rendering inside to ensure that formatting of the text is done in one place.

Reply:
I thought about this and there might be some benefits in forcing the title to be a String and not just a random component. I added a Pill component in categoryPill.js